README.md

```diff
@@ -1,8 +1,8 @@
 ### Current Build Status
 
-|Travis | Appveyor | Read the Docs |
-| --- | --- | --- |
-| [![travis_ci](https://travis-ci.org/pyFFTW/pyFFTW.svg?branch=master)](https://travis-ci.org/pyFFTW/pyFFTW) | [![appveyor_ci](https://ci.appveyor.com/api/projects/status/uf854abck4x1qsjj/branch/master?svg=true)](https://ci.appveyor.com/project/hgomersall/pyfftw) | [![read_the_docs](https://readthedocs.org/projects/pyfftw/badge/?version=latest)](http://pyfftw.readthedocs.io/en/latest/?badge=latest) |
+| GitHub Actions                                                                                                  | Read the Docs                                                                                                                           |
+| --------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| ![github_actions](https://img.shields.io/github/workflow/status/JeppeKlitgaard/pyFFTW/wheel_tests_and_release)  | [![read_the_docs](https://readthedocs.org/projects/pyfftw/badge/?version=latest)](http://pyfftw.readthedocs.io/en/latest/?badge=latest) |
```

Comment:
Presumably this needs updating to be the correct githubs action badge? (i.e. not yours)

Reply:
Yes, absolutely. I think this change is better made by you directly to `master` after this is merged, as the configuration we want should target `master` and it is not possible to really test before this is merged.